ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly-4.16-upgrade-from-stable-4.15.yaml

```diff
@@ -573,6 +729,59 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-nutanix-ipi-proxy
+- as: nutanix-ipi-proxy-fips-minimal-permission-f28
```

Comment:
If we decide to use the `minimal-permission` install workflow to run upgrade, pls remove the above one - `nutanix-ipi-proxy-fips-f28`, it is duplicated.